devdocs/godmode.md

```diff
@@ -0,0 +1,77 @@
+#GodMode User Notes
+
+##What is GodMode?
+
+Typically browser tests involve comparing the source of the webpage to an existing *expected* source. This serves to verify the DOM structure (HTML) of the page. However, creating new browser tests and updating outdated ones require considerable effort on the developer's part. In addition, ensuring that the generated *expected* source code works across computers, browsers, user accounts, and execution times is harder still. To overcome this problem, GodMode provides a simple way to create and update *expected* source files for browser tests and ensure the necessary cross-compatibility.
+
+
+##How does GodMode work?
+
+The essential idea is to reverse the process of testing, whereby the actual source of the webpage during the test is used to overwrite the *expected* source for the test, as opposed to the usual way of using the *expected* source as the reference and checking the actual source of the webpage. In actual implementation we check if the test already passes and only if it fails do we update the *expected* source. Thus running GodMode on passing tests poses no danger.
+
```

Comment:
Any simpler way to explain this. e.g. We use the 'actual' to overwrite the 'expected' automatically and then manually look through the diff to ensure the new 'expected' is as we intended.